spring-cloud-kubernetes-fabric8-config/src/test/java/org/springframework/cloud/kubernetes/fabric8/config/LabeledConfigMapContextToSourceDataProviderTests.java

```diff
@@ -411,4 +416,56 @@ void searchWithLabelsTwoConfigMapsFoundAndOneFromProfileFound() {
 
 	}
 
+	/**
```

Comment:
a test that proves that log statements are on par with caching, first request is from k8s, subsequent ones are from cache